### DISCUSSION TITLE
tsh login does not open browser window

### DISCUSSION THREAD

**Discussion by ZeroClover** [gravitational/teleport#7065] [Q&A] [answered]
When I log in using `tsh login`, tsh asks me to enter the user password in the terminal instead of opening a browser page.

The only way I can get tsh to open the browser for SSO login by adding the `--auth=github` parameter.

However, according to the Teleport documentation, it seems that tsh should always open the browser for authentication unless the `--browser none` parameter is added.

Is there a problem with my configuration? Or is this just how it was designed?

**Comment by yjperez** [comment, marked as answer]
@ZeroClover apologies we missed this one. You can set your default auth type, this would prevent you from having to using the flag at login.

On your auth server :
```
auth_service:
    enabled: yes
    authentication:
        type: github
```
